modules/ROOT/examples/Columnar.py

```diff
@@ -2,14 +2,14 @@
 
 # tag::init[]
 spark = (SparkSession.builder
```

Comment:
Do we also want to include an example of connecting to Enterprise Analytics?  And if so, should that link to the existing SDK/server docs (not sure where it ended up) on hosts + ports.